lib/pavilion/plugins/commands/status.py

```diff
@@ -255,20 +333,31 @@ def _setup_arguments(self, parser):
             '--history', type=int,
             help="Shows the full status history of a job."
         )
+        parser.add_argument(
+            '--series', type=str, default=False, nargs=2,
+            help='Displays test series based on user specifications.'
+        )
         parser.add_argument(
             '-s', '--summary', default=False, action='store_true',
             help='Summary will display a summed version of what'
                  'state were observed in pav status. '
         )
```

Comment:
What happened to filtering by test name, user, and host (sys_name)?

Reply:
You told me to just do '--user' and '--sys_name`. I can add those if you want.